packages/web-react/src/components/FileUploader/AttachmentDismissButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classNames from 'classnames';
+import { SpiritAttachmentDismissButtonProps } from '../../types';
+import { Icon } from '../Icon';
+import { useFileUploaderStyleProps } from './useFileUploaderStyleProps';
+import { useStyleProps } from '../../hooks';
+
+const AttachmentDismissButton = (props: SpiritAttachmentDismissButtonProps) => {
+  const { children, ...restProps } = props;
+
+  const { classProps } = useFileUploaderStyleProps();
+  const { styleProps, props: transferProps } = useStyleProps(restProps);
+
+  return (
+    <button
+      type="button"
+      {...transferProps}
+      {...styleProps}
+      className={classNames(classProps.attachment.button, styleProps.className)}
+    >
+      <span className="accessibility-hidden">{children}</span>
```

Comment:
I think we might want to have a default value? I think in Twig we have it as a prop with default.

Reply:
Default value is set in `FileUploaderAttachment.tsx`. We will replace this button with the global close button